mapsforge-map-reader/src/main/java/org/mapsforge/map/reader/MapFile.java

```diff
@@ -1072,6 +1072,11 @@ public boolean supportsTile(Tile tile) {
                 && (tile.zoomLevel >= this.zoomLevelMin && tile.zoomLevel <= this.zoomLevelMax);
     }
 
+    @Override
+    public boolean supportsArea(BoundingBox boundingBox) {
+        return boundingBox.intersects(getMapFileInfo().boundingBox);
```

Comment:
This should also contain the zoom level check like `supportsTile`.

Reply:
My initial ideal was that zoom level should not play the role here, but as `MapFile` has it's defined limits, you are of course correct that it has to be there as well. I'll improve it.